models/articles.models.js

```diff
@@ -16,3 +16,20 @@ exports.selectArticleById = (requestedArticleId) => {
       return article;
     });
 };
+
+exports.adjustArticleVotes = (requestedArticleId, voteCount) => {
+  return db
+    .query(
+      `UPDATE articles SET votes = votes + ${voteCount} WHERE article_id = ${requestedArticleId} RETURNING *;`,
+    )
+    .then(({ rows }) => {
+      const article = rows[0];
+      if (!article) {
+        return Promise.reject({
+          status: 404,
+          msg: `No article found for article_id: ${requestedArticleId}`,
```

Comment:
Is there a test for this?
I'm only seeing whats in this pull request, so can only see the test for the patch stuff.